builders/build/prerequisites.md

```diff
@@ -0,0 +1,86 @@
+---
+
+# Prerequisites for Building a ContainerChain {: #prerequisites } 
+
+## Introduction {: #introduction } 
+
+Deploying a ContainerChain through Tanssi is a fairly straightforward step, where the only requirement is to have a valid [chain specification](https://docs.substrate.io/build/chain-spec/){target=_blank} to upload to the Tanssi network and make it go live.
+
+To generate a Substrate chain specification, it is necessary to have a development environment where a Substrate node can be compiled, and, to do so, the minimal required software and its installation process will be covered in the next sections of this article.
+
+## Rust {: #rust } 
+
+[Rust](/learn/framework/overview/#rust-programming-language) is a modern, portable, and performant programming language that is the base of the Substrate blockchain development framework.  
+
+To compile the Appchain, the rust compiler, *rustc*, and the package manager, *cargo*, must be installed on the system. 
+
+### Installing Rust via *rustup* {: #install-via-rustup } 
+
+For any system running Linux or MacOS, the following command will do:
+
+=== "Linux"
+    ```bash
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+    ```
+=== "MacOS"
+    ```bash
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+    ```
+
+When the installation process is completed, running the following command verifies that the newly installed compiler works correctly by showing the version number:
+
+=== "Linux"
+    ```bash
+    rustc --version
+    ```
+=== "MacOS"
+    ```bash
+    rustc --version
+    ```
+
+There are other methods to install Rust, such as using a package manager. Other options can be found on the [Rust official site](  https://forge.rust-lang.org/infra/other-installation-methods.html){target=_blank}.
+
+## Installing Git {: #installing-git } 
+
+Git is recommended to clone the [code repository](https://github.com/moondance-labs/tanssi){target=_blank} of Tanssi, where the node templates can be found. Git is likely shipped within the default OS installation configuration or included in other tools, such as Xcode in MacOS.
+
+If Git is not present on the system, the following command will install it using a package manager:
+
+=== "Linux (Ubuntu/Debian)"
+
+    ```bash
+    apt-get install git
+    ```     
+=== "MacOS"
+
+    ```bash
+    brew install git
+    ```     
+
+## Checking the Installation {: #checking-installation } 
+
+With these essential tools installed, the development environment is ready to compile the Tanssi node or one of the included templates.
+
+The following commands build the EVM-compatible template and generate the chain specification:
+
+1. Clone the Tanssi code hosted on GitHub
+```bash
+git clone https://github.com/moondance-labs/tanssi
+```
+2. Step into the project folder
+```bash
+cd tanssi
+```
+3. Build the EVM-compatible Appchain template
```

Comment:
Link to the templates page? It doesn't have to be the EVM template, right?

Reply:
We have currently 2 templates, but 99% of the teams work with EVM, so it covers the most representative case